tests/e2e/taqueria-plugin-flextesa.spec.ts

```diff
@@ -0,0 +1,242 @@
+            dockerName = "local"
+
+            // 1. Run sandbox start command
+            const stdoutStart = execSync(`taq start sandbox`, {cwd: `./${taqueriaProjectPath}`}).toString().trim();
+
+            // 2. Verify that sandbox has been started and taqueria returns proper message into console
+            expect(stdoutStart).toEqual("Started local.");
+
+            // 3. Verify that docker container has been started
+            const [,dockerContainerLocal] = execSync(`docker ps --filter "name=${dockerName}"`).toString().trim().split(/\r?\n/);
+            expect(dockerContainerLocal).toBeDefined();
+
+            // 4.  Run stop command and verify the output
+            const stdoutStop = execSync(`taq stop sandbox ${dockerName}`, {cwd: `./${taqueriaProjectPath}`}).toString().trim();
+
+            // 5. Verify that taqueria returns proper message into console
+            expect(stdoutStop).toEqual("Stopped local.");
+
+        } catch(error) {
+            throw new Error (`error: ${error}`);
+        }
+
+    });
+
+    test('Verify that taqueria flextesa plugin can start and stop a default "local" sandbox', async () => {
+        try {
+
+            // Setting up docker container name
+            dockerName = "local"
+
+            // 1. Run sandbox start command
+            const stdoutStart = execSync(`taq start sandbox ${dockerName}`, {cwd: `./${taqueriaProjectPath}`}).toString().trim();
+
+            // 2. Verify that sandbox has been started and taqueria returns proper message into console
+            expect(stdoutStart).toEqual("Started local.");
+
+            // 3. Verify that docker container has been started
+            const [,dockerContainerLocal] = execSync(`docker ps --filter "name=${dockerName}"`).toString().trim().split(/\r?\n/);
+            expect(dockerContainerLocal).toBeDefined();
+
+            // 4. Verify that sandbox started on the proper port 0.0.0.0:20000->20000/tcp
+            let isReachable = await isPortReachable(20000, {host: 'localhost'});
+            expect(isReachable).toBeTruthy();
+
+            // 4.  Run stop command and verify the output
+            const stdoutStop = execSync(`taq stop sandbox ${dockerName}`, {cwd: `./${taqueriaProjectPath}`}).toString().trim();
+
+            // 5. Verify that taqueria returns proper message into console
+            expect(stdoutStop).toEqual("Stopped local.");
+            isReachable = await isPortReachable(20000, {host: 'localhost'})
+            expect(isReachable).toBeFalsy();
+
+        } catch(error) {
+            throw new Error (`error: ${error}`);
+        }
+
+    });
+
+    test('Verify that taqueria flextesa plugin can return list of accounts from a sandbox', async () => {
+        try {
+
+            // Setting up docker container name
+            dockerName = "local"
+
+            // 1. Run sandbox start command
+            execSync(`taq start sandbox`, {cwd: `./${taqueriaProjectPath}`}).toString().trim();
```

Comment:
If you're starting a sandbox in this test should you not also stop it as a clean up step?

Reply:
Yes and afterEach( ()...) does it for me. It runs after each test to tackle the clean-up process.